doc/source/examples.rst

```diff
@@ -30,6 +30,59 @@ follows.
 
 .. _`quay.io`: https://quay.io/repository/ansible/molecule
 
+Docker With Non-Privileged User
+===============================
+
+Default Molecule Docker driver execute Ansible playbooks with root user. If your
+workflow require non-privileged user, some simple changes to ``molecule.yaml``
```

Comment:
Just curious: what could require such use-case?

Reply:
In our organisation after initial configuration `root` user is disabled so `sudo` is only way. So if I don't test with `become: yes` it is only partial test since later I need to add `become` to some tasks. We are not only one after some short googling [random example](https://www.rubytreesoftware.com/resources/ansible-bootstrap-playbook/) there is more.

Reply:
Okay, fair enough. Initially, I thought it's about the image we ship with the molecule itself :)